academy-of-management-review.csl

```diff
@@ -99,7 +102,14 @@
                 </date>
               </if>
             </choose>
-          </if>
+          </else-if>
+          <else-if variable="issued" match="any">
+            <choose>
```

Comment:
unless I'm missing something, you don't need to test for status here. just use the current line 109.